aws/aws_instance.go

```diff
@@ -226,12 +228,23 @@ func (awsI *Ec2Instance) runTerraformApply(volumeSize int, zookeeperSize int, ma
 		return err
 	}
 
+	if (bastionVolSnapshotId != "") {
```

Comment:
Ideally this check would be near the logic that places all the other assets in newAwsDeploymentDescription.  Is that possible?  If not can we move it up to CreateInstance()?